### ISSUE TITLE
Ability to override prefers-reduced-motion media query

### ISSUE THREAD

**Issue by numanzahid** [elementor/elementor#9726]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**

Elementor implemented the `prefers-reduced-motion` query in the main CSS. Which is really good but there should be an option to override this. You can make this on by default but you should allow us to override this in the settings.

There was confusion about animation not working in the community. many of the user still don't know the cause. People still think it is a bug.

https://github.com/elementor/elementor/issues/8508

you might say that it is for accessibility, but it is not fair to force accessibility features on the developer. You should allow developer to switch this setting off. 

Please don't mind me saying; Reducing animation is one thing related to accessibility. there are so much more to that. like colors and fonts and font sizes. What are you going to do? simply stop everything through your code? block use of some colors, fonts or font sizes? I don't think so. It is the designer who has to think about these problems.

**Describe the solution you'd like**
Allow switching off or on `prefers-reduced-motion media query`

**Describe alternatives you've considered**
At least provide us some code that can be used in function file or CSS to override `prefers-reduced-motion`. That will be really helpful.

**Additional context**
Please don't take this request wrong. I am not against accessibility and stuff.
If I have stopped animation it does not necessarily mean I am allergic to animations. It is very common practice the stop OS animation to make it look fast.
If you quick search internet you will be amazed to know how much people stop OS animation for speed optimization. and this practice is common across platforms, Linux, windows and Mac. People even stop animations on their phones, not for the accessibility, but for speed. It is not the indication that they don't want to see animation on my webpage.

**Comment by drawcard** [comment]
I guess everyone's philosophy about web design is different, but one thing pretty much all developers agree on is that the web should be accessible to as many as possible. All devs should be making it their mission, [particularly in the face of growing lawsuits](https://www.essentialaccessibility.com/blog/web-accessibility-lawsuits/) over the issue. 

There are many out there who are photosensitive and have had seizures triggered by animations, so it's for that reason things like `prefers-reduced-motion` exist. If developers are using it to speed up their computer to use Elementor, that's not what `prm` is about and Elementor shouldn't be expected to compensate for the incorrect use of accessibility features. 

If a toggle is implemented, it should enable `prm` CSS by default, to allow the accessibility feature to work as intended, for users who expect it to work in line with their browser & OS settings.

**Comment by numanzahid** [comment]
@drawcard 
Thanks for the detailed response. I am not asking to remove this function or remove it by default. It should be on naturally, that will be a good practice. but there should be an option to switch it off.

Please don't isolate reduce motion through all accessibility options. there are multiple design elements that need to be considered by the front end designer. reduce motion is one thing. A platform like elementor allows designer to apply those features as he sees fit. it can help designer to make all those amendments, but it can not force designer to implement. 

there are things like font size, dark mode, users who can not use mouse, color contrast and many more. elementor can encourage designer to use all these features, but it can not stop if the designer do otherwise. I don't think elementor have a way to forcefully implement all these settings. what do you think? if not then why reduce motion is exception? Why it can not be a choice like many other accessibility features?

Again, don't get me wrong. I am not against implementing all accessibility settings, but it is a choice.

**Comment by zeinnicholas** [comment]
It is already possible to disable this using css. You just need to find the declaration and override it. But honestly? You shouldn't. And you know why you shouldn't. But if you want to go that route, then be that guy...

**Comment by arielk** [comment]
I'm closing an issue due to inactivity. If the problem still exists, feel free to reopen the issue or contact our support through your account. 

Thanks!

**Comment by rami-elementor** [comment]
- Spec: https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion
- MDN: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion
- Web.dev: https://web.dev/prefers-reduced-motion/

**Comment by rami-elementor** [comment]
So I checked our code and it seems that we already support `prefers-reduced-motion` for over 4 years (since Elementor 2.6).

Source: https://github.com/elementor/elementor/blob/master/assets/dev/scss/frontend/_animate.scss#L24-L29

The solution was sudgested by @drawcard in https://github.com/elementor/elementor/pull/7968/files
